pfcpiface/utils.go

```diff
@@ -99,3 +102,26 @@ func maxUint64(x, y uint64) uint64 {
 func calcBurstSizeFromRate(kbps uint64, ms uint64) uint64 {
 	return uint64((float64(kbps) * 1000 / 8) * (float64(ms) / 1000))
 }
+
+// GetUnicastAddressFromInterface returns a unicast IP address configured on the interface.
+func GetUnicastAddressFromInterface(interfaceName string) (net.IP, error) {
+	iface, err := net.InterfaceByName(interfaceName)
+	if err != nil {
+		log.Errorln("Unable find interface with name", interfaceName, err)
```

Comment:
since this is a utility function and we print error in the caller (`NewUPF()`), would it make sense to skip logging error inside this function?